modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
```

Comment:
make this say:  Configure autoscaling to dynamically scale the number of worker nodes up or down based on load. Autoscaling for worker nodes is configured in the machine pool definition.

Reply:
Done.